install.sh

```
@@ -1,28 +1,53 @@
+readonly CHECKSUM_FILE="$SCRIPT_DIR/Installers/.checksums.sha256"
+readonly EXIT_APP_CODE=42
+
+# --- 3. COMMAND AVAILABILITY CACHE ---
+HAS_GIT=$(command -v git &>/dev/null && echo 1 || echo 0)
+HAS_SHA256SUM=$(command -v sha256sum &>/dev/null && echo 1 || echo 0)
+HAS_FILE=$(command -v file &>/dev/null && echo 1 || echo 0)
+HAS_SUDO=$(command -v sudo &>/dev/null && echo 1 || echo 0)
+HAS_TIMEOUT=$(command -v timeout &>/dev/null && echo 1 || echo 0)
+
+# --- 4. STATIC SYSTEM INFO CACHE ---
+readonly CACHED_HOSTNAME=$(hostname)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)